teacher.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
+    @specialization = specialization
+    @name = name
+    super(age, parent_permission: parent_permission)
+  end
```

Comment:
- [ ] Name should be included in the `super` call because we also want to get the name from the parent class.